src/async.cpp

```diff
@@ -49,13 +49,38 @@ static int fuse_getattr(const char *path, struct stat *stbuf, struct fuse_file_i
 	int pid;
 	char p[32];
 	int ret = sscanf(path, "/%d/%s", &pid, p);
+
 	if(ret == 1) {
 		// we only matched "/1000"
 		stbuf->st_mode = S_IFDIR | 0755;
 		stbuf->st_nlink = 0;  // TODO correct value
 		return 0;
 	}
 	if(ret == 2) {
+		char f[32];
+		if(sscanf(path, "/%d/fdinfo/%s", &pid, f) == 2) {
```

Comment:
Aren't you checking for `/1000/fdinfo/name` here instead of  `/1000/fdinfo/$fd/name` here?

Reply:
ah :laughing: i got it!
> sscanf(path, "/%d/fdinfo/%s/%s", &pid, b, f)

I think the correct one **should** be
```
sscanf(path, "/%d/fdinfo/%d/%s", &pid, &fd, f)
```

Reply:
Works like a charm!! I wonder why I haven't tried this before 🤦🏻